bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableVeneerApi.java

```diff
@@ -63,5 +84,18 @@ public DataClientWrapper getDataClient() {
   public void close() throws IOException {
     dataClientWrapper.close();
     adminClientWrapper.close();
+    activeSessions.dec();
+    for (int i = 0; i < channelPoolSize; i++) {
+      BigtableClientMetrics.counter(MetricLevel.Info, "grpc.channel.active").dec();
+    }
```

Comment:
Doesnt this need to account for channel pooling as well? ie defer decrementing to the factory?

Reply:
actually nevermind, channelPoolSize will be 0